### PR TITLE
Add chectl cli tests in OCP 4.5 4.6 4.7(nightly) jobs

### PR DESCRIPTION
**Pull request by flacatus**
Add latest OCPs version to test chectl.

Refference Issue: https://github.com/eclipse/che/issues/18293
Signed-off-by: Flavius Lacatusu <flacatus@redhat.com>

### CODE REVIEW

**Comment by flacatus** [comment]
Pending to merge: https://github.com/che-incubator/chectl/pull/998 to have tests green.

File: ci-operator/config/che-incubator/chectl/che-incubator-chectl-master__v7.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+base_images:
+  cli:
+    name: "4.6"
```
Comment (tolusha) [inline ci-operator/config/che-incubator/chectl/che-incubator-chectl-master__v7.yaml:3]: 4.7 ?

**Review by tolusha — APPROVED** [review]

**Pull request merged**